client/i18n/locales/english/intro.json

```diff
@@ -3926,8 +3926,8 @@
         ]
       },
       "lecture-understanding-abstraction": {
-        "title": "Understanding abstraction",
-        "intro": ["Learn about Understanding abstraction in these lectures."]
+        "title": "Understanding Abstraction",
```

Comment:
was a decision made about this? there were talks about removing the abstraction content

Reply:
for now it is in the sheet, we can remove it later if needed, but maybe @Dario-DC has an answer already